res/packet-capture.yml

```diff
@@ -31,6 +31,22 @@ spec:
             value: "false"
           - name: LOG_LEVEL
             value: info
+          - name: INTERFACES
+            value: ""
+          - name: EXCLUDE_INTERFACES
+            value: "lo"
+          - name: SAMPLING
+            value: "1"
+          - name: ENABLE_RTT
+            value: "false"
+          - name: ENABLE_PKT_DROPS
+            value: "false"
+          - name: ENABLE_DNS_TRACKING
+            value: "false"
+          - name: ENABLE_NETWORK_EVENTS_MONITORING
+            value: "false"
+          - name: ENABLE_FLOW_FILTER
+            value: "false"
```

Comment:
do we need those changes since those features are irrelevant to pca ? just wanted to avoid any confusion if someone mistakenly enabled them ?

Reply:
sure let's get rid of these

Reply:
https://github.com/netobserv/network-observability-cli/pull/90/commits/06068df3ee304ea3e372ba591be489dc339fac23